CMakeLists.txt

```diff
@@ -685,15 +685,21 @@ else()
   include_directories(Externals/mbedtls/include)
 endif()
 
-find_package(CURL)
-if(CURL_FOUND)
-  message(STATUS "Using shared libcurl")
-  include_directories(${CURL_INCLUDE_DIRS})
+# The libcurl we currently use has a harcoded certificate location
+# It can only be used on distributions where SSL certificates actually are at that path.
+if (NOT UNIX OR EXISTS "/etc/ssl/certs/ca-certificates.crt")
```

Comment:
This checks whether the file exists on the system that is building Dolphin, right? That's going to be a problem when cross-compiling for Android.

Reply:
I guess I could check [CMAKE_CROSSCOMPILING](https://cmake.org/cmake/help/latest/variable/CMAKE_CROSSCOMPILING.html)?